scripts/sysman/powerLimits.yml

```diff
@@ -108,6 +114,7 @@ details:
     - "The same limit can be supplied multiple times. Limits are applied in the order in which they are supplied."
     - "The application may call this function from simultaneous threads."
     - "The implementation of this function should be lock-free."
+    - "1.7": "Note: This function is deprecated and functionality is provided by $sPowerSetPowerLimits."
```

Comment:
`- "1.7": "[DEPRECATED] Use $sPowerSetPowerLimits."`

Reply:
need to define new set of interfaces first